src/30_Advanced/Advanced_Video_Docking.html

```diff
@@ -0,0 +1,223 @@
+    }
+
+    header {
+      background: white;
+      padding: 10px 20px;
+      border-bottom: 1px solid #ddd;
+      position: fixed;
+      left: 0;
+      right: 0;
+      top: 0;
+      z-index: 1;
+      line-height: 40px;
+      font-size: 18px;
+    }
+
+    main {
+      margin: 0 auto;
+      padding: 80px 0 0 0;
+    }
+
+    .content {
+      height: 200vh;
+      width: calc(70vw - 20px);
+      padding: 0 20px;
+      box-sizing: border-box;
+    }
+
+    aside {
+      right: 20px;
+      margin-left: 70vw;
+      position: fixed;
+      top: 80px;
+
+      /*
+      Define a viewport-based width for the aside. The docking slot will expand
+      to fill its width.
+      */
+      width: 30vw;
+    }
+
+    .more-content {
+      padding: 20px 20px 0;
+      background: #eaeaea;
+
+      /*
+      Offset vertical axis by the negative height of the docking slot.
+      Since the docking slot is 30vw wide, its height is 16.85vw based on a 16:9
+      aspect ratio
+      */
+      transform: translateY(-16.85vw);
+    }
+
+    .more-content ul {
+      margin: 0;
+      padding: 0;
+      list-style: none;
+    }
+
+    .more-content li {
+      display: block;
+      margin: 0 -20px;
+    }
+
+    .more-content li a {
+      display: block;
+      border-top: 1px solid #ddd;
+      text-decoration: none;
+      padding: 16px 20px;
+      color: #2ac;
+    }
+
+    h3 {
+      margin: 0;
+      padding-bottom: 20px;
+    }
+  </style>
+</head>
+
+<body>
+  <!-- ## Define animations -->
+  <!-- To slide down the content below the docking slot, the vertical offset is
+  changed into a 20px positive offset to create padding below the slot. -->
+  <amp-animation layout="nodisplay" id="slideDown">
+    <script type="application/json">
+      {
+        "selector": ".more-content",
+        "duration": "0.5s",
+        "fill": "both",
+        "easing": "ease-out",
+        "keyframes": [{
+            "offset": 0,
+            "transform": "translateY(-16.85vw)"
+          },
+          {
+            "offset": 1,
+            "transform": "translateY(20px)"
+          }
+        ]
+      }
+    </script>
+  </amp-animation>
+  <!-- To slide up the content once the video is no longer docked, the animation
+  reverts the aside content's offset to -16.85vw. -->
+  <amp-animation layout="nodisplay" id="slideUp">
+    <script type="application/json">
+      {
+        "selector": ".more-content",
+        "duration": "0.5s",
+        "fill": "both",
+        "easing": "ease-out",
+        "keyframes": [{
+            "offset": 0,
+            "transform": "translateY(20px)"
+          },
+          {
+            "offset": 1,
+            "transform": "translateY(-16.85vw)"
+          }
+        ]
+      }
+    </script>
+  </amp-animation>
+  <header>My Header</header>
+  <main>
+    <div class="content">
+      <!-- ## Include your video -->
+      <!-- In your `<amp-video>` tag, you should include two attributes:
```

Comment:
Can you link `amp-video` to the reference doc.

Reply:
Done, the same with `amp-animation`.